### PR TITLE
fix editbox placehold height bug

### PR DESCRIPTION
**Pull request by novain1102**
Re: # https://github.com/cocos/cocos-engine/issues/16040

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: cocos/ui/editbox/edit-box.ts [hunk 1]
```diff
@@ -759,7 +759,6 @@ export class EditBox extends Component {
 
         if (placeholderLabel) {
             placeholderLabel.node._uiProps.uiTransformComp!.setContentSize(size.width - LEFT_PADDING, size.height);
-            placeholderLabel.lineHeight = size.height;
```
Comment (novain1102) [inline cocos/ui/editbox/edit-box.ts:762]: It should assign from label's lineHeight attribute from editor directly like textLabel. No need to assign value here.

Comment (minggo) [reply cocos/ui/editbox/edit-box.ts:762]: If it is not set in editor. Will it also work correctly?

Comment (novain1102) [reply cocos/ui/editbox/edit-box.ts:762]: > If it is not set in editor. Will it also work correctly?

Sure. label component has initial value when created.

Comment (novain1102) [reply cocos/ui/editbox/edit-box.ts:762]: > It should assign from label's lineHeight attribute from editor directly like textLabel. No need to assign value here.

And there's no meaningful to assign placeholderLabel.lineHeight = size.height;

**Review by minggo — APPROVED** [review]

**Pull request merged**